xml/geo_manage_i.xml

```diff
@@ -114,6 +127,14 @@ booth[27900]: 2014/08/13_10:21:23 info: revoke succeeded!</screen>
    instead. Find the exact status in the log files or with the
    <command>crm_ticket -L</command> command.
   </para>
+  <para>
+   The <command>grant</command> command can use a <option>-F</option> option.
+   This option forces a site to get a ticket even if the ticket is granted
```

Comment:
I would rephrase to: 'This option is potentially dangerous, but it can be useful for manual ticket management in case a site which holds a ticket cannot be reached. It makes booth grant the ticket *immediately* to the specified site, even though booth cannot ascertain that other sites (that cannot be reached) do not hold the same ticket.' 

I would also add a warning:
'WARNING Concurrency Violation and Potential Loss of Data
Before you use this option, make sure that no other site currently owns the same ticket. If the same ticket is granted to multiple sites, resources depending on the ticket might be started on multiple sites in parallel, resulting in concurrency violation and potential data corruption.'